web/src/admin/applications/wizard/application/ApplicationStep.ts

```diff
@@ -0,0 +1,25 @@
+import { type WizardButton } from "@goauthentik/components/ak-wizard-main/types";
```

Comment:
This is the individual step, broken out into its own file and placed next to the renderer.  The same is true for ProviderChoice, Provider, and Submit.